architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,58 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
+//from https://blog.openshift.com/image-streams-faq/
+An _image stream is a pointer to an image using various identifying qualities. This means that even if the source image changes, 
+the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+Note that an image stream can be configured to periodically chack for updates (using the `--scheduled` flag on a tag) 
+or you can re-importing an image (using the `oc import-image` command). 
+
+An image stream does not contain actual image data, it is simply a mapping between tags and actual images. 
+It can contain additional metadata that describes the tags provided by the image stream.
+
+The images can be stored in any of the following:
+
+. {product-title}'s xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+registry]
+. An external registry, such as `registry.access.redhat.com`, `hub.docker.com`, and so forth.
+. Other image streams
+
 An _image stream_ comprises any number of
 xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
 container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+related images, similar to an image repository. 
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
-registry]
-. Other image streams
-. Image repositories from external registries
+==== Important terms 
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry: *docker.io/openshift/jenkins-2-centos7:latest*.
+
+Docker image tag::
+A  label applied to a Docker image in a repository that distinguish specific images. For example, here *latest* is a tag:
+*docker.io/openshift/jenkins-2-centos7:latest*
+
+Docker image sha::
```

Comment:
s/Docker image sha/Docker image id

SHA is a shorthand for secure hashing algorithm, but the name most frequently use is ID here.